lib/src/tasks/task_runner/api.dart

```diff
@@ -0,0 +1,118 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:dart_dev/util.dart' show reporter, TaskProcess;
+
+import 'package:dart_dev/src/tasks/task.dart';
+
+Future<TaskRunner> runTasks({List<String> tasksToRun}) async {
```

Comment:
This param shouldn't be optional since you're not defaulting it to anything in that scenario.